react/details/package.json

```diff
@@ -1,24 +1,28 @@
 {
-  "name": "conference-details-widget",
+  "name": "all-typed-blob-entity-details-widget",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@entando/apimanager": "^4.1.0",
```

Comment:
we're relying on fetch instead of apiManager

Reply:
forgot I have added that one, removed